user_docs/en/userGuide.t2t

```diff
@@ -2231,6 +2231,25 @@ This setting may stabilize reading of incoming text in some applications.
 However, in terminals, when inserting or deleting a character in the middle of a line, the text after the caret will be read out.
 -
 
+==== Speak new text in Windows Terminal via ====[WtStrategy]

```

Comment:
Does this feature refer to new text, or is it actually text changes? Diffing and "report dynamic content changes" makes me think this is text changes rather than new text.